### PR TITLE
add wallet reconnected snackbar

### PR DESCRIPTION
**Pull request by zielvna**

### CODE REVIEW

File: src/store/sagas/wallet.ts [hunk 1]
```diff
@@ -139,14 +139,32 @@ export function* handleAirdrop(): Generator {
...
+          message: 'Wallet reconnected.',
+          variant: 'info',
+          persist: false
+        })
+      )
+    } else {
+      yield* put(
+        snackbarsActions.add({
+          message: 'Wallet connected.',
+          variant: 'info',
```
Comment (p6te) [inline src/store/sagas/wallet.ts:162]: for wallet connect probably better looks like 'success' variant

**Review by p6te — COMMENTED** [review]
There is still missing functionality for this task. 
We want to show the snackbar with reconnect button when user change the account in wallet extension. 
1. connect the wallet.
2. change account in wallet extension
image
3. Show snackbar - wallet reconnected, with button to reconnect wallet on invariant

File: src/containers/HeaderWrapper/HeaderWrapper.tsx [hunk 2]
```diff
@@ -22,14 +22,11 @@ export const HeaderWrapper: React.FC = () => {
...
       const wallet = await getAlephZeroWallet()
-      wallet.addListener('connect', () => {
-        dispatch(walletActions.connect())
-      })
 
       await wallet.canEagerConnect().then(
         async canEagerConnect => {
           if (canEagerConnect) {
-            dispatch(walletActions.connect())
+            dispatch(walletActions.connect(true))
```
Comment (p6te) [inline src/containers/HeaderWrapper/HeaderWrapper.tsx:29]: apply these changes to all 'connect wallet' buttons in the app

**Review by p6te — APPROVED** [review]

**Review by wojciech-cichocki — CHANGES_REQUESTED** [review]
I don't know if this error occurred with this PR or if it exists on dev, but there is a problem with 'change wallet'. Below is a video:
[Screencast from 18.07.2024 11:21:43.webm](https://github.com/user-attachments/assets/6967f3f0-8209-4df1-a70c-ea73292eb329)

**Comment by p6te** [comment]
after refreshing the page wallet is not reconnected.
[Nagranie ekranu z 19.07.2024 08:56:14.webm](https://github.com/user-attachments/assets/ccb01bed-867a-4248-b472-d006a96d3363)

**Comment by p6te** [comment]
> after refreshing the page wallet is not reconnected. [Nagranie ekranu z 19.07.2024 08:56:14.webm](https://github.com/user-attachments/assets/ccb01bed-867a-4248-b472-d006a96d3363)

it works correctly on localhost

**Review by p6te — APPROVED** [review]

**Review by wojciech-cichocki — CHANGES_REQUESTED** [review]
In my case it doesn't work

**Review by wojciech-cichocki — APPROVED** [review]

**Pull request merged**